toolchain/check/check.cpp

```diff
@@ -231,8 +254,10 @@ static auto BuildApiMapAndDiagnosePackaging(
       continue;
     } else if (import_key.first == CppPackageName) {
       CARBON_DIAGNOSTIC(CppPackageDeclaration, Error,
-                        "`Cpp` cannot be used by a `package` declaration");
-      unit_info.emitter.Emit(packaging->names.node_id, CppPackageDeclaration);
+                        "`{0}` cannot be used by a `package` declaration",
+                        std::string);
+      unit_info.emitter.Emit(packaging->names.node_id, CppPackageDeclaration,
+                             CppPackageName.str());
```

Comment:
```suggestion
                        "`{0}` cannot be used by a `package` declaration",
                        llvm::StringLiteral);
      unit_info.emitter.Emit(packaging->names.node_id, CppPackageDeclaration,
                             CppPackageName);
```

Reply:
As discussed, StringLiteral isn't supported so moved back to `Cpp`.